src/screens/ChannelListScreen/elements/CustomPreviewAvatar.js

```diff
@@ -49,7 +50,7 @@ const CustomPreviewAvatar = ({channel}) => {
           source={{uri: channel.data.image, priority: FastImage.priority.normal}}
           style={styles.image}
         />
-        <View style={styles.typeContainer('#55C2FF')}>
+        <View style={styles.typeContainer(COLORS.blueSea)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to a dynamic one from the theme object is a good practice for maintainability and consistency. However, it's important to ensure that `COLORS.blueSea` exists in your theme object and it's the correct color you want to use.
```diff
- <View style={styles.typeContainer('#55C2FF')}>
+ <View style={styles.typeContainer(COLORS.blueSea)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->